modules/5_install/install.tf

```diff
@@ -70,6 +70,15 @@ locals {
         log_level               = var.log_level
         release_image_override  = var.release_image_override
         rhcos_kernel_options    = var.rhcos_kernel_options
+        sysctl_tuned_options    = var.sysctl_tuned_options
+        sysctl_options          = var.sysctl_options
+        match_array_info        = [ for ix in range(length(var.match_array)) :
```

Comment:
Do we really need to process the data?

Reply:
Yes, we need to process it to have uniform map objects for better templating later.